documentation/config_file_logbackxml.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: default
+title: "Puppet Server Configuration Files: logback.xml"
+canonical: "/puppetserver/latest/config_file_logbackxml.html"
+---
+
+Puppet Server’s logging is routed through the Java Virtual Machine's (JVM) [Logback library](http://logback.qos.ch/). By default, it logs to `/var/log/puppetlabs/puppetserver/puppetserver-access.log`, the log level is ‘INFO’, and Puppet Server sends nothing to `syslog`.
```

Comment:
Oh, I see, you cover the HTTP request logging below.  So it's probably just a typo on the file path here then.